pkg/kubelet/dockertools/docker_test.go

```diff
@@ -653,7 +652,7 @@ func TestFindContainersByPod(t *testing.T) {
 	fakeClient := NewFakeDockerClient()
 	np, _ := network.InitNetworkPlugin([]network.NetworkPlugin{}, "", nettest.NewFakeHost(nil), componentconfig.HairpinNone, "10.0.0.0/8")
 	// image back-off is set to nil, this test should not pull images
-	containerManager := NewFakeDockerManager(fakeClient, &record.FakeRecorder{}, nil, nil, &cadvisorapi.MachineInfo{}, options.GetDefaultPodInfraContainerImage(), 0, 0, "", &containertest.FakeOS{}, np, nil, nil, nil)
+	containerManager := NewFakeDockerManager(fakeClient, &record.FakeRecorder{}, nil, nil, &cadvisorapi.MachineInfo{}, "", 0, 0, "", &containertest.FakeOS{}, np, nil, nil, nil)
```

Comment:
The tests in this commit originally depended on some code in `k8s.io/kubernetes/cmd/kubelet/app/options/options.go`, and now that code has moved to `k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1/defaults.go`. @mikedanese and I discussed whether we should just make the tests depend on `defaults.go`, or reduce the maintenance burden by passing empty strings here. 

We determined that:
1. Changing the tests in this way didn't cause any of them to fail, so we aren't presently deriving coverage from having an accurate image name here. We therefore don't lose coverage with this change, but we do (very slightly) reduce the dependency tree, which is good. 
2. We will know if a future test requires an accurate image name, because such a test will fail without it. We can therefore drop the dependency without fear of accidentally missing coverage down the road. 

Given these conditions, we figured it makes the most sense to just pass `""` here.